### DISCUSSION TITLE
Welcome to courses Discussions!

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/courses#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
![IMG_20230512_191347](https://github.com/SkalskiP/courses/assets/132863086/51a592ed-49a3-4658-9590-53138076193f)
![FB_IMG_1683356449262](https://github.com/SkalskiP/courses/assets/132863086/9d9a94e1-ac3d-4ad8-bd62-f811c9abc8ce)
![IMG-20230424-WA0000](https://github.com/SkalskiP/courses/assets/132863086/3e2c20e5-b718-4ca1-b18b-8f752dcfa55c)
I would like to ask for anybody opinion about either resetting the entire technology and financial system to where there isn't near as much confusion and no more of those ads every 5 seconds
